packages/prisma/migrations/20231001101010_add_user_email_to_booking_time_status/migration.sql

```diff
@@ -0,0 +1,35 @@
+-- View: public.BookingTimeStatus
+
+-- DROP VIEW public."BookingTimeStatus";
+
+CREATE OR REPLACE VIEW public."BookingTimeStatus"
+ AS
+ SELECT "Booking".id,
+    "Booking".uid,
+    "Booking"."eventTypeId",
+    "Booking".title,
+    "Booking".description,
+    "Booking"."startTime",
+    "Booking"."endTime",
+    "Booking"."createdAt",
+    "Booking".location,
+    "Booking".paid,
+    "Booking".status,
+    "Booking".rescheduled,
+    "Booking"."userId",
+    et."teamId",
+    et.length AS "eventLength",
+        CASE
+            WHEN "Booking".rescheduled IS TRUE THEN 'rescheduled'::text
+            WHEN "Booking".status = 'cancelled'::"BookingStatus" AND "Booking".rescheduled IS NULL THEN 'cancelled'::text
+            WHEN "Booking"."endTime" < now() THEN 'completed'::text
+            WHEN "Booking"."endTime" > now() THEN 'uncompleted'::text
+            ELSE NULL::text
+        END AS "timeStatus",
+    et."parentId" AS "eventParentId",
+    "u"."email" AS "userEmail",
+    "u"."username" AS "username"
+   FROM "Booking"
+     LEFT JOIN "EventType" et ON "Booking"."eventTypeId" = et.id
+     LEFT JOIN users u ON u.id = "Booking"."userId";
```

Comment:
Removed unused membership left join here and added new users to get user->email and username